shared/chat/conversation/input-area/normal/mention-input.js

```diff
@@ -221,13 +241,19 @@ class MentionInput extends React.Component<MentionInputProps, MentionState> {
 
   _forceSubmit = () => {
     const text = this._getText()
+    const {lastSelectionKey} = this.state
```

Comment:
...and then here the check could be `if (text && this._lastPickSelectedKey === 'Enter')`

Reply:
Actually, perhaps rename this function to `_maybeSubmit` or `_maybeSubmitAfterInsert`